### PR TITLE
Feature: Getting amenities

### PR DESCRIPTION
**Pull request by brianmuse**
This PR adds endpoints to `GET` space and organization amenities.

### CODE REVIEW

File: src/main/java/com/robinpowered/sdk/model/Amenity.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package com.robinpowered.sdk.model;
+
+import com.google.common.base.Objects;
+import org.joda.time.DateTime;
+
+/**
+ * An amenity that may be associated to a {@link Space}.
```
Comment (christophermark) [inline src/main/java/com/robinpowered/sdk/model/Amenity.java]: Associated _with_?

File: src/main/java/com/robinpowered/sdk/model/Amenity.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    public static final String MIME_TYPE = "vnd.robinpowered.amenity.v1";
+
+
+    /**
+     * Properties
+     */
+
+    // Immutable
+    private final Integer id;
```
Comment (christophermark) [inline src/main/java/com/robinpowered/sdk/model/Amenity.java]: Is there a reason why we have non-null properties as nullable objects?
Could this be `int id` instead?

File: src/main/java/com/robinpowered/sdk/model/Calendar.java [hunk 3]
```diff
@@ -12,7 +12,7 @@
  * automatically be synced to the space.
  *
  * <p>
- * Spaces without calendars are considered "breakout" spaces are not able to be reserved.
+ * Spaces without calendars are considered "on demand" spaces are not able to be reserved.
```
Comment (christophermark) [inline src/main/java/com/robinpowered/sdk/model/Calendar.java]: *on-demand

File: src/main/java/com/robinpowered/sdk/model/Amenity.java (hunk 2 above)
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/Amenity.java]: You're right. `int` is better here

File: src/test/java/com/robinpowered/sdk/model/AmenityTest.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.robinpowered.sdk.model;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.junit.Test;
+
+public class AmenityTest {
+
+    @Test
+    public void testEqualsAndHashcode() {
+        EqualsVerifier.forClass(Amenity.class)
```
Comment (christophermark) [inline src/test/java/com/robinpowered/sdk/model/AmenityTest.java:10]: Did you intend to delete this class?

File: src/main/java/com/robinpowered/sdk/model/Amenity.java (hunk 1 above)
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/Amenity.java]: I think either is OK, but I'll change. dont feel strongly.

File: src/test/java/com/robinpowered/sdk/model/AmenityTest.java (hunk 4 above)
Comment (brianmuse) [inline src/test/java/com/robinpowered/sdk/model/AmenityTest.java:10]: What do you mean? This is a unit test for the new model.

Comment (christophermark) [inline src/test/java/com/robinpowered/sdk/model/AmenityTest.java:10]: My bad, I missed the Test directory. Ignore this comment

**Comment by christophermark** [comment]
Looks good! :+1:

**Pull request merged**